views/default/widgets/elgg_news/rss.js

```diff
@@ -0,0 +1,50 @@
+define(['jquery', 'elgg'], function ($, elgg) {
+	
+	return function(selector, feed_url) {
+		var $wrapper = $(selector);
+		var wrapper_id = $wrapper.attr("id");
+		$wrapper.empty();
+		
+		var config = $wrapper.data();
+		
+		var feed_url = config.feedUrl;
+		var limit = config.limit;
+
+		$.getJSON(
+			"//ajax.googleapis.com/ajax/services/feed/load?v=1.0&num=" + limit + "&output=json_xml&q=" + encodeURIComponent(feed_url) + "&hl=en&callback=?",
```

Comment:
This goes away in a few months. https://developers.google.com/feed/v1/jsondevguide